src/utils/bbox.hpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * DeepDetect
+ * Copyright (c) 2014-2015 Emmanuel Benazera
```

Comment:
I don't remember writing this :) I let you update the copyright and email!

Reply:
Sure :)